concordium-consensus/src/Concordium/GlobalState/Block.hs

```diff
@@ -343,7 +343,8 @@ instance forall pv. (IsProtocolVersion pv) => BlockData (Block pv) where
             STOV1 -> emptyTransactionOutcomesHashV1
     blockTransactionOutcomesHash (NormalBlock bb) = blockTransactionOutcomesHash bb
 
-    -- FIXME: replace stub, and move into gendata
+    -- Note. Do not rely on this for type class for retrieving the block state hash of a
+    -- genesis block. Instead use @bpBlockStateHash@ of 'BlockPointerData'.
     blockStateHash GenesisBlock{} = StateHashV0 minBound
     blockStateHash (NormalBlock bb) = blockStateHash bb
 
```

Comment:
Do we need this method then at all?

Reply:
We don't usually solve problems like this, by having a function we write to not use.

Reply:
Actually, I think we can remove it without causing too much disturbance.